### ISSUE TITLE
PROPERTYPUT fails due to invalid number of parameters expected.

### ISSUE THREAD

**Issue by alexkoay** [mhammond/pywin32#1427]
Running on Python 3.7 with pywin32 225.

I'm trying to interface with SAP GUI Scripting read-write properties, but writing to properties gives me an error as it doesn't do PUTs correctly. I've traced the issue back to commit to fix #1199 where the return value is changed from `invoke_type` to `varkind`. I believe some kind of special typing should be done with regards to PUTs instead of just sending back `varkind` directly.

The error I'm encountering:
```
File "\lib\site-packages\win32com\client\dynamic.py", line 559, in __setattr__
    self._oleobj_.Invoke(entry.dispid, 0, invoke_type, 0, value)
pywintypes.com_error: (-2147352562, 'Invalid number of parameters.', None, None)
```

**Comment by mhammond** [comment]
Sorry for the delay - I've started looking at this and things are a bit of a mess around this code. I've added [some comments](https://github.com/mhammond/pywin32/blob/ea7c9a6dab73451b2c2d5dcdc8866b7b4fa84ff7/com/win32com/client/dynamic.py#L100) about that and I'm slowly cleaning things up a little so I can re-understand how things work :) It might take me a bit longer to untangle and I've a few other commits queued up in that direction. Please let me know if you are able to help me test things when I get a little further along - thanks!

**Comment by mhammond** [comment]
(oops - that was actually meant for #1641, but it's still relevant here, so 👍 )

**Comment by mhammond** [comment]
I think I fixed this in af54b27e8b9b04fe5937a78a1aaeba2eecaca824 - please reopen if you still have problems after the next release.
